### ISSUE TITLE
Update for 2016

### ISSUE THREAD

**Issue by vbudhram** [vbudhram/fantasy-football-io#2]
Currently, ESPN and Yahoo parsing is broken for ~~2015~~2016.

**Comment by JamesTeague** [comment]
Is this still an issue? at least pertaining to this year. I was looking to use this API but maybe I can look into helping fix the problem.

**Comment by vbudhram** [comment]
Hey @JamesTeague, unfortunately I have not really had any time to dedicate to this. The API has not been updated to this year and is probably broken. More than welcome to take a stab at updating for 2016 :)

**Comment by devotionsolutions** [comment]
Is it still broken?

**Comment by JamesTeague** [comment]
@blumersolutions Yes, I originally started working on the ESPN fix but due to ESPN's new method of logging in, I was not able to make too much progress there. I haven't really had too much time either to work on this but it is still on my backlog of things to look at.

I do believe I did some work around some of the news feeds but I will have to look at my branches and refresh myself on what I did.

**Comment by devotionsolutions** [comment]
Thanks for the info. I'll keep watching this project :)
